tests/sentry/replays/endpoints/test_project_replay_summary.py

```diff
@@ -262,7 +265,7 @@ def test_post_with_both_direct_and_trace_connected_errors(
         assert any("Failed to connect to database" in log for log in logs)
 
     @patch("sentry.replays.endpoints.project_replay_summary.make_signed_seer_api_request")
-    def test_post_with_feedback(self, mock_make_seer_api_request: Mock) -> None:
+    def test_post_with_feedback_breadcrumb(self, mock_make_seer_api_request: Mock) -> None:
         """Test handling of breadcrumbs with user feedback"""
```

Comment:
```suggestion
        """Test handling of a feedback breadcrumb when the feedback is in nodestore, but hasn't reached Snuba yet. If the feedback is in Snuba, it should be de-duped like in the duplicate_feedback test below."""
```
This is how I'm interpreting this test, this explanation could be helpful

Reply:
guaranteed for sdk v8+

Reply:
ok updated the comment!